docs/api.md

```diff
@@ -89,6 +90,34 @@ Sets the _folder name_ for the working directory of your WebContainer instance.
 
 This is mostly a "cosmetic" option.
 
+#### `forwardPreviewErrors?: boolean | 'exceptions-only'`
+
+Configure whether errors occurring in embedded preview iframes should be forwarded to the parent page. Captured errors originate from:
+
+ - Calls to `console.error`
+ - Any `unhandledrejection` events on `window`
+ - Any uncaught `error` events on `window`
+
+If set to `'exceptions-only'`, `console.error`s are not forwarded.
+
+Default value is `false`, so no errors are emitted.
+
+To receive the events, you register an event handler like this:
+
+```js
+import { isPreviewMessage } from '@webcontainer/api';
+
+window.addEventListener('message', (event) => {
+  const data = event.data;
+
+  if (!isPreviewMessage(data)) {
+    return;
+  }
+
+  // process the data received from a preview
```

Comment:
```suggestion
  // process the data received from the preview
```

Reply:
Actually I think `a` is better because events can come from any preview, there isn't necessarily only one (in TutorialKit for instance you can have more than one preview).

Reply:
Oh, I suggested `the` cause you add the event listener on a specific `window`. Is that `window` the preview iframe? I am a tad confused 😂

Reply:
Ah no worries! No it's not the preview iframe, it's the global `window` object defined in the user's page where they import the `@webcontainer/api` package.

Reply:
Yea then `a` makes sense.